.drone.yml

```diff
@@ -608,6 +616,9 @@ steps:
     build_args:
     - ARCH=arm64
     - VERSION=${DRONE_BRANCH/release\//}-${DRONE_COMMIT}-head
+    - CATTLE_RANCHER_WEBHOOK_VERSION=104.0.0+up0.5.0-rc8
```

Comment:
Rather than adding the hardcoded values here, can we instead use `build_args_from_env` and maybe use `command:` or something similar to `source scripts/package-env`?

Reply:
Let me look into Drone's docs for that. This would be a much better fix - a release captain won't need to update these values then.